fred2/sexp_tree.cpp

```diff
@@ -3519,6 +3519,12 @@ int sexp_tree::query_default_argument_available(int op, int i)
 			}
 			return 0;
 
+		case OPF_TRAITOR_OVERRIDE:
+			if (Traitor_overrides.size() > 0) {
```

Comment:
Here and in qtfred, `empty()` could be used instead of `size() > 0`.  Same effect either way, just a style thing,

Reply:
Style/consistency? Who cares? It's a free for all.

```
case OPF_ASTEROID_DEBRIS:
			if ((Asteroid_info.size() - NUM_ASTEROID_POFS) > 0) {
				return 1;
			}
			return 0;

		case OPF_MOTION_DEBRIS:
			if (Motion_debris_info.size() > 0) {
				return 1;
			}
			return 0;

		case OPF_BOLT_TYPE:
			if (Bolt_types.size() > 0) {
				return 1;
			}
			return 0;

		case OPF_TRAITOR_OVERRIDE:
			return Traitor_overrides.empty() ? 0 : 1;
```

Reply:
True enough.